pkg/services/action.go

```diff
@@ -244,8 +241,7 @@ func NewSubmitBusinessCase(
 		updatedAt := config.clock.Now()
 		businessCase.UpdatedAt = &updatedAt
 
-		if businessCase.SystemIntakeStatus == models.SystemIntakeStatusBIZCASEFINALNEEDED ||
-			intake.Step == models.SystemIntakeStepFINALBIZCASE {
+		if intake.Step == models.SystemIntakeStepFINALBIZCASE {
```

Comment:
double check

Reply:
this needs to be looked at as it changes a conditional

Reply:
This looks right.

Reply:
Agreed. This looks good. I'll resolve!